packages/calypso-e2e/src/lib/pages/my-home-page.ts

```diff
@@ -2,6 +2,9 @@ import { Page } from 'playwright';
 
 const selectors = {
 	visitSiteButton: '.button >> text=Visit site',
+
+	// Task card (topmost card)
+	taskMessage: ( message: string ) => `div.task h2:has-text("${ message }")`,
```

Comment:
Generic selector that will match the heading item shown in My Home dashboard.

Reply:
In the function comments, you say it checks for a string on the card, but this selector limits it to just the header. I think we should clarify either way and lock that in more cleary!

Reply:
I thought I had specified it was only the header. I'll review and update the comments as necessary.